projects/goharbor/harbor/docker/linux/Dockerfile

```diff
@@ -0,0 +1,33 @@
+ARG BASE_IMAGE # https://gallery.ecr.aws/eks-distro-build-tooling/eks-distro-base
+FROM $BASE_IMAGE
+
+ARG RELEASE_BRANCH
+ARG TARGETARCH
+ARG TARGETOS
+
+RUN yum install -y shadow-utils >> /dev/null \
+    && yum clean all \
+    && groupadd -f -r -g 10000 chart \
+    && useradd --no-log-init -m -g 10000 -u 10000 chart
+
+COPY _output/$RELEASE_BRANCH/dependencies/$TARGETOS-$TARGETARCH/eksa/helm/chartmuseum/chartm /home/chart
+COPY ../harbor/make/photon/chartserver/docker-entrypoint.sh /home/chart/
+COPY ../harbor/make/photon/common/install_cert.sh /home/chart/
+COPY _output/LICENSES /LICENSES
+COPY ATTRIBUTION.txt /ATTRIBUTION.txt
+
+RUN chown -R chart:chart /etc/pki/tls/certs \
+    && chown -R chart:chart /home/chart \
+    && chmod u+x /home/chart/chartm \
+    && chmod u+x /home/chart/docker-entrypoint.sh \
+    && chmod u+x /home/chart/install_cert.sh
+
+USER chart
+
+WORKDIR /home/chart
+
+ENTRYPOINT ["./docker-entrypoint.sh"]
+
+VOLUME ["/chart_storage"]
```

Comment:
We never really add these calls, not sure if it is necessary or not since kubernetes is going to override them anyway?